gitpod-io/website
# [ref-arch/eks] Remove clusterissuer namespace

## Description

ClusterIssuer objects aren't namespaced; this commit removes the extraneous metadata value from the EKS ClusterIssuer setup.

## How to test

Apply the modified ClusterIssuer to a pre-existing cluster and install Gitpod, checking that certificates are issued as expected.


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2735"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
